class.jetpack-gutenberg.php

```diff
@@ -717,4 +719,83 @@ public static function block_classes( $slug = '', $attr, $extra = array() ) {
 
 		return implode( ' ', $classes );
 	}
+
+	/**
+	 * Determine whether a site should use the default set of blocks, or a custom set.
+	 * Possible variations are currently beta, experimental, and production.
+	 *
+	 * @since 8.1.0
+	 *
+	 * @return string $block_varation production|beta|experimental
+	 */
+	public static function blocks_variation() {
+		if ( Constants::is_true( 'JETPACK_BETA_BLOCKS' ) ) {
+			return 'beta';
+		}
+
+		/*
+		 * Switch to experimental blocks if you use the JETPACK_EXPERIMENTAL_BLOCKS constant.
+		 */
+		if ( Constants::is_true( 'JETPACK_EXPERIMENTAL_BLOCKS' ) ) {
+			return 'experimental';
```

Comment:
Instead of returning early, could we set a variable and return only after the filter? As it is now, if beta or experimental constant is set, there is no way to filter that value.

Reply:
Good call! Done in 258350b3308d121096e55e43dec1d9f35177aa33